frontend/src/types.ts

```diff
@@ -478,6 +478,12 @@ export interface DashboardType {
     tags: string[]
 }
 
+export interface DashboardRefreshStatusType {
```

Comment:
This type isn't used anywhere

Reply:
Nice catch, forgot to remove this. I wish `dashboardLogic.js` was typescriptified, which I'll probably tackle in a future PR.